pycalculix/base_classes.py

```diff
@@ -8,11 +8,13 @@
             - 'stress': stress
             - 'strain': strain
             - 'force': force
+            - 'error': FE discretization error in stress
         value (list):
             - 'displ' = 'ux','uy','uz','utot'
             - 'stress' = 'Sx','Sy','Sz','Sxy','Syz','Szx','Seqv','S1','S2','S3'
             - 'strain' = 'ex','ey','ez','exy','eyz','ezx','eeqv','e1','e2','e3'
             - 'force' = 'fx','fy','fz'
+            = 'error' = 'STR(%)'
```

Comment:
Can we change this to e% to be consistent with our strain nomenclature?
Or strain% to be fully explicit?
Let's also keep this docstring consistent with our code below.